e2e/tests/dashboard-tests.spec.ts

```diff
@@ -1,35 +1,15 @@
 import { expect, test } from "#e2e/fixtures.ts";
 
-test.describe("Dashboard Module", () => {
```

Comment:
No need to wrap everything in a test file. This flattens the structure a bit and makes it more readable.